03/app.js

```diff
@@ -43,16 +43,46 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
+    if(typeof this[lang] === undefined){
+        return null;
+    }
+
+    if(lang === 'pl'){
+        const titlePl = this[isbn]['title']['pl'];
+        return titlePl;
+    }
+
+    if(lang ==='en'){
+        const titleEn = this[isbn]['title']['en'];
+        return titleEn;
+    }
```

Comment:
To rozwiązanie jest wygodne tylko dla 2 języków - co jeśli będzie ich 100?
Trzeba to rozwiązać bardziej elastycznie czyli:
```
   const title = this[isbn]['title'][lang]
    if(typeof title !== 'undefined'){
        return title;
    }
```